examples/react/start-workos/app/authkit/ssr/workos.ts

```diff
@@ -0,0 +1,39 @@
+import { WorkOS } from '@workos-inc/node';
+import { getConfig } from './config';
+import { lazy } from './utils';
+
+const VERSION = '0.0.1';
```

Comment:
Is this related to a WorkOS SDK version? or the user application version?

A comment would help.

Reply:
I'll remove this. I put this together from an existing SDK and plan to make a more generic _SSR_ library to build these SDKs off of.